.github/dependabot.yml

```diff
@@ -68,11 +68,29 @@ updates:
   - mtoonen
   - mprins
   labels:
-  - dependencies
   - javascript
   - build infrastructure
   - backport v5.8.x
 
+- package-ecosystem: npm
+  directory: "/tailormap-components"
+  schedule:
+    interval: weekly
+    day: wednesday
+    timezone: Europe/Amsterdam
+  open-pull-requests-limit: 25
+  reviewers:
+    - mtoonen
+    - geertplaisier
+  assignees:
+    - mtoonen
+    - geertplaisier
+  labels:
+    - dependencies
+    - javascript
+    - build infrastructure
+    - backport v5.8.x
```

Comment:
Zou dit niet standaard doen, volgens mij geeft dit meer problemen dan het gaat oplossen: backwards compatibility van de tailormap-components lijkt mij erg laag

Reply:
dat betekent dat er geen security updates van dependencies worden gedaan in de stable branch

Reply:
Hmm, true... misschien heeft @geertplaisier of @matthijsln hier nog een mening over?

Reply:
Lijkt me niet nodig

Reply:
kennelijk vindt ook niemand het nodig om de master branch up2date te houden wat betreft NPM dependencies, gezien het aantal updates waarvoor nu PR aangemaakt worden door dependabot nadat deze PR is gemerged

Reply:
Elke keer als je je omdraait is er wel weer een nieuwe update van een library, vaak met een sub-minor versie. DIt kost teveel tijd en zorgt voor ruis. Updates neem ik meestal mee als ik een groot iets doe. Misschien zou een maandelijks in te plannen libraryupdate-taakje in jira beter werken?